modules/kubernetes-addons/aws-vpc-cni/data.tf

```diff
@@ -0,0 +1,19 @@
+data "aws_iam_policy_document" "ipv6_policy" {
```

Comment:
can add statement to deploy only if ipv6 is required

Reply:
updated